benchmarks/spectral_analysis.py

```diff
@@ -0,0 +1,120 @@
+import matplotlib.pyplot as plt
+import gammapy
+import numpy as np
+import astropy
+import regions
+import astropy.units as u
+from astropy.coordinates import SkyCoord, Angle
+from regions import CircleSkyRegion
+from gammapy.maps import Map
+from gammapy.utils.fitting import Fit
+from gammapy.data import ObservationStats, ObservationSummary, DataStore
+from gammapy.background import ReflectedRegionsBackgroundEstimator
+from gammapy.spectrum.models import PowerLaw
+from gammapy.spectrum.crab import MeyerCrabModel
```

Comment:
You can remove the MeyerCrabModel import

Reply:
done